### ISSUE TITLE
When selecting and deselecting tasks afterwards the status is still success

### ISSUE THREAD

**Issue by inf17101** [kentaro-m/task-completed-checker-action#64]
When marking all tasks as completed in the PR and afterwards deselecting some of the task, then the task-list-completed-checker is triggered again but is success (green).

Expected: The PR shall not be allowed to merge if a task is deselected

**Steps to reproduce:**
1. .github/workflows/task-list-checker.yml
2. Put the following config in:
```
name: 'PR Tasks Completed Check'
on: 
  pull_request:
    types: [opened, edited, reopened, synchronized]
  
jobs:
  task-check:
    permissions: write-all
    runs-on: ubuntu-latest
    steps:
      - uses: kentaro-m/task-completed-checker-action@v0.1.2
        with:
          repo-token: "${{ secrets.GITHUB_TOKEN }}"
```
3. Create a PR request template .github/pull_request_template.md with the following dummy content:
```

- [ ] task 1
- [ ] task 2
- [ ] task 3
```
4. Open a PR
5. Wait for initial completion of task-list-completed-checker
6. Mark all three tasks as done
7. Wait for the next completion of task-list-completed-checker again
8. Mark one task as "not complete" again
9. Wait for the next completion of task-list-completed-checker again
